mail_tracking/models/mail_message.py

```diff
@@ -203,16 +203,16 @@ def get_failed_messages(self):
                 for msg in self.sorted('date', reverse=True)]
 
     @api.multi
-    def toggle_tracking_status(self):
-        """Toggle message tracking action.
+    def set_need_action_done(self):
+        """Set message tracking action as done
 
         This will mark them to be (or not) ignored in the tracking issues
         filter.
         """
         self.check_access_rule('read')
-        self.mail_tracking_needs_action = not self.mail_tracking_needs_action
+        self.mail_tracking_needs_action = False
         notification = {
-            'type': 'toggle_tracking_status',
+            'type': 'mail_tracking_needs_action_changed',
```

Comment:
I would let the notification type the same, for being able to reuse if we want to notify also a new error in the future.

Reply:
Forget it. I'm seeing that the name is more specific and cover both cases.